x-pack/plugins/security/public/views/logged_out/components/logged_out_page.tsx

```diff
@@ -35,7 +35,7 @@ export class LoggedOutPage extends Component<Props, {}> {
             </div>
           </header>
           <div className="loggedOut__content eui-textCenter">
-            <EuiButton href={this.props.addBasePath('/login')}>
+            <EuiButton href={this.props.addBasePath('/')}>
```

Comment:
optional: would it be clearer to just use `chrome.getBasePath()` (or a similar `this.props.getBasePath()`, since the new intent is to navigate to Kibana's root?